### DISCUSSION TITLE
Cash Flow Management

### DISCUSSION THREAD

**Discussion by ScottFries** [ghostfolio/ghostfolio#2651] [Q&A] [answered]
I'm not sure if I'm missing something or this is just an odd omission, but it doesn't seem like Ghostfolio has any concept of cash flow in/out of accounts. How are you expected to represent (cash) transfers between accounts such as:

1. Adding money to a brokerage account
2. Withdrawling money from a brokerage account
3. Transferring stock shares between accounts
4. Transferring cash between accounts

**Comment by dtslvr** [comment, marked as answer]
Hi @ScottFries

There is no concept for deposit / withdrawal in Ghostfolio. However, you can update the cash balance of any account (e.g. when you receive your salary).

1. Go to _Accounts_ and click _Transfer Cash Balance_
2. Go to _Accounts_ and click _Transfer Cash Balance_
3. Update manually (edit the account of the activities)
4. Go to _Accounts_ and click _Transfer Cash Balance_

**Reply by ScottFries** [reply]
Thanks for confirming! Is that an intentional design decision, or just absent because it hasn't proved to be necessary? It seems to me like it'd block features like importing transaction history from external sources and accurately reporting total profit/loss.

**Reply by dtslvr** [reply]
The second, not present because it has not proved to be necessary.

**Reply by diamant-x** [reply]
Given the inclusion of https://github.com/ghostfolio/ghostfolio/issues/2491 wouldn't it make sense to support deposit/withdrawal? Its easier to track when money is added or removed, than to keep track everyday of the balance for that day. Then let Ghostfolio do the maths to show the account balance for each historical day.
